### DISCUSSION TITLE
2022-07-20 Weekly Meeting 65 Agenda - Home Spaces

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#254] [Weekly Meetings]
Weekly Meeting 65 is happening on Wednesday July 20th, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda
- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers
- Check in on MSF. Let's check in on a plan for them to a. submit OMI to MSF, and b. how they will manage the delgate roles (checking email, passing along information, attending MSF meetings as OMI rep, etc.)
- Reminder: our next official demo day occurring during our Wednesday meeting on July 27th  

### Working Group Updates:
- [OMI Website update](https://github.com/omigroup/website)
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - OMI developer hour
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- any others?

#### Upcoming Wednesday Calendar `themes`: :calendar: (continued in the [omigroup/media repo](https://github.com/omigroup/media/discussions/43))
- Last week we discussed `Portals`, t- Today's topic was suggested to be `Homespaces` What could interoperable homespace templating look like? Jin suggested doing workshops, and discussions on VTubing, and more. (July 20, 2022)

![image](https://user-images.githubusercontent.com/63426722/180029598-ce85286a-5fb8-4117-87d1-593fd789653f.png)

#### Future
- OMI DEMO DAY `Open Metaverse`  (July 27, 2022) - [sign up here](https://docs.google.com/forms/d/e/1FAIpQLSfwrdJQ4hk-nc9t9-qG_UfCkzTQE1Cd1QyLVMMs4y9asLdnmQ/viewform)
- `content portability`  (August 3, 2022) 
- `OMI-audio-emitter` exploration and demo 
- `OMI-collider` exploration and demo

**Comment by mrmetaverse** [comment]
# OMI and MSF 
- #250 - submitted OMI to MSF 
- need to create a new msf-delegate@omigroup.org email 
- would like to see MSF delegate volunteers set a date to meet and decide how they would like to proceed/divide up work.

**Comment by mrmetaverse** [comment]
For demos today we have : 

- Jin 
- TBD

**Comment by mrmetaverse** [comment]
# Shoutout M3! 

https://twitter.com/dankvr/status/1548048469301944320

![image](https://user-images.githubusercontent.com/63426722/180031889-a993f0e8-a440-46fa-a3b9-8ec5a3b0dfe5.png)

**Reply by mrmetaverse** [reply]
Donations like this open up possibilities for OMI. Most of our budget has gone to cloud services and tools we use as a community. But if you have a collaborative jam idea with another community, or an experiment in your working group, please contact OMI leads about getting a budget for your experiments!

**Comment by mrmetaverse** [comment]
# Scripting group 
- exploring the scope of the group 
- if interested, reach out! Tell us what your project needs related to user generated content or sandbox scripting! Post in our github https://github.com/omigroup/omi-scripting-group
- particularly interested in individuals with Web Assembly experience, making these APIs friendly for use grounded in where WASM is at currently. 
- Next scripting group meeting in 2 weeks 
- Next week is the next OMI developer hour! 
  - want to demo during that time? 
  - techy, in the weeds, blocked, engineering-wise? This is the place for that

**Comment by mrmetaverse** [comment]
# gLTF 
- got KHR audio edits all done 
- submitted into PR against the Khronos repo
  - ready for feedback on [this PR]() now 
  - seems like the draft to get our final feedback on!  
  - next meeting is tomorrow 21:30UTC (2:30 PM PST)

**Reply by mrmetaverse** [reply]
https://gestaltor.io/2022/07/18/gestaltor-2022-3-0/

**Comment by mrmetaverse** [comment]
# Media group 
- 1password blocker
  - need to fix youtube to ensure we can stream in time for next week 
  - anything we want to do to promote the stream differently?
- Red will help promote in Reddit and other channels 
- If you see screenshots during demos that are worth tweeting, send them our way!

**Comment by mrmetaverse** [comment]
# Demos!

**Comment by mrmetaverse** [comment]
## Jin and the core of home space interop 
- [XR Devlog](https://hackmd.io/@XR/openmetaverse) 
- The keys to your house, do you actually own them? 
- M3 website, everything you see is a link
- ![image](https://user-images.githubusercontent.com/63426722/180035697-367d4ae4-67fc-4e9a-beae-22ab2c2f672d.png)
- ![image](https://user-images.githubusercontent.com/63426722/180035780-68b0699f-ca40-44bb-831d-d50c9d53fa82.png)
- ![image](https://user-images.githubusercontent.com/63426722/180035854-e62d4275-3bbc-4e0a-9551-0683268d0d26.png)
- ![image](https://user-images.githubusercontent.com/63426722/180035871-1795e3f5-11e5-42a2-b96a-19dac6010170.png)
- ![image](https://user-images.githubusercontent.com/63426722/180036050-c0897d5f-004b-42cd-8603-a50609a407ea.png)
- Jin demos using exploring a home space inside of a home space, inside of a home space :scream: 
- ![image](https://user-images.githubusercontent.com/63426722/180036163-1f6f9ddb-6c38-460b-a776-955b29a5cc4a.png)
- ![image](https://user-images.githubusercontent.com/63426722/180036314-0f735151-189a-4cc1-9566-d4dfe7e8b67c.png)
- >The dream is a space that is truly yours, that you can self-host, open source, invite friends over, and play anything on the internet. Stream your desktop, do LAN parties, and more all in your own virtual world. 
- >I think of the shipping container as the carriage wagons in the Wild West, exploring different platforms. I want to bring stuff with me and my stuff isn't all going to fit in a backpack. 
- Explore for yourself on m3org.com
- Newest experiments have been reusing the clonex pod across multiple platforms. 
- ![image](https://user-images.githubusercontent.com/63426722/180036834-686b76b5-c558-41ac-beb3-936c4a82f7e0.png)
- We want avatars to be interoperable, so shouldn't the home spaces be as well? 
- Jin added a shell to it to bring it into other Worlds 
- ![image](https://user-images.githubusercontent.com/63426722/180036935-9acefa1c-b402-4fc5-b341-5f1e032d986a.png)
- ![image](https://user-images.githubusercontent.com/63426722/180036987-cd99b940-a9dd-411f-8c6d-5d00bf0310f2.png)
- and here it is in Anarchy Arcade as well. 
- ![image](https://user-images.githubusercontent.com/63426722/180037122-91a81b93-8674-4de8-ad98-318f9d3b6ffe.png)
- layer 1 
![image](https://user-images.githubusercontent.com/63426722/180037141-e5b9c64c-88be-4700-b563-a05e3e88b0a4.png)
- layer 2
![image](https://user-images.githubusercontent.com/63426722/180037161-cc61b990-e6bc-4e9b-a636-615efd75691e.png)
- layer 3 
![image](https://user-images.githubusercontent.com/63426722/180037190-2c221c15-2b8c-4f96-a5da-caa7f0c52f36.png)
- ported into layer 4 
![image](https://user-images.githubusercontent.com/63426722/180037221-70b3c1b7-7168-40b9-b8ce-582631776203.png)
- portaled in from a poster into layer 5 
![image](https://user-images.githubusercontent.com/63426722/180037266-c6fc80fb-e4fa-45dc-b199-bedac865a900.png)
- create a town or district by remixing and stacking homespaces 
![image](https://user-images.githubusercontent.com/63426722/180037360-7684b9df-d38b-4c14-a293-9767a40d6b2a.png)
- Jin dropped a link to watch SithLords twitch stream on a tv :tv: 
![image](https://user-images.githubusercontent.com/63426722/180037704-052314ea-67cf-4c5d-9bb9-586e9fe283e5.png)
- ![image](https://user-images.githubusercontent.com/63426722/180037788-05d379ff-8a2f-480b-aee5-e0aa3e13970e.png)
- ![image](https://user-images.githubusercontent.com/63426722/180037889-2b05f74b-c561-48a8-b132-f091e9b3d4c0.png)
- Jin talked about the significance of home spaces not being device locked or platform locked in. 
- ![image](https://user-images.githubusercontent.com/63426722/180038042-332779b2-1e2b-4589-aa22-dcf6869e80e8.png)
![image](https://user-images.githubusercontent.com/63426722/180038052-bc632aaf-69e0-4c5a-8d63-92d7213e930f.png)
- multiple demos in different platforms like Janus and Neos 
- ![image](https://user-images.githubusercontent.com/63426722/180038218-8179a7a8-913f-41b5-9ae9-bfa14d18baf1.png)
- Substrata VR can pull in parcels from Voxels too 
- ![image](https://user-images.githubusercontent.com/63426722/180038297-e528362e-f13b-4161-b925-b3ea819f30be.png)
- Somnium 
![image](https://user-images.githubusercontent.com/63426722/180038327-a61c2b6b-0485-416f-97a2-631c842c9313.png)
- VR Chat 
![image](https://user-images.githubusercontent.com/63426722/180038369-df55524c-19fb-44d6-9999-ad065c711381.png)
![image](https://user-images.githubusercontent.com/63426722/180038442-fd54bbee-c86e-4b63-bce2-b3f2a8536316.png)
- Webaverse 
- ![image](https://user-images.githubusercontent.com/63426722/180038502-8fa7c14a-38f1-472d-8453-3cd514e92c0d.png)
- so many more, I couldn't keep up! 
- Q: are they all GLBs? 
- A: Sometimes, other times they are voxels. I didn't have to use FBX. Getting into Anarchy Arcade required some modifications. 
- A cool thing about Anarchy Arcade is I can bring this with me to other worlds. 
  - ![image](https://user-images.githubusercontent.com/63426722/180038733-647c6a47-e469-474d-84f2-7d183b8501b9.png)
  - ![image](https://user-images.githubusercontent.com/63426722/180038807-ff688357-9324-4d74-bbda-f60b2ce3c356.png)
  - you can bring it with you! 
  - ![image](https://user-images.githubusercontent.com/63426722/180038989-b8d73b47-0ebf-4ca4-adf3-b3dbc0be34d1.png)
  - again... do you really own it if its tied to one platform or device? 
  - ![image](https://user-images.githubusercontent.com/63426722/180039119-754e3e6c-c8f2-4688-b794-f13766e356b0.png)
- M3 will make Jins shell for the CloneX pods available to all holders to make their homespaces more portable. 
- ![image](https://user-images.githubusercontent.com/63426722/180039280-cf4ee135-a10f-4dfd-ae50-7fc46bfe980b.png)

## TO get more involved 
- ![image](https://user-images.githubusercontent.com/63426722/180039636-575c08df-12b4-4bfb-ace9-d0aa1a5793ee.png)
- :link: https://github.com/madjin/homespace

**Reply by mrmetaverse** [reply]
Hit Jin up to hack on this stuff more!

**Comment by mrmetaverse** [comment]
Sign up for [our next demo day here](https://docs.google.com/forms/d/e/1FAIpQLSfwrdJQ4hk-nc9t9-qG_UfCkzTQE1Cd1QyLVMMs4y9asLdnmQ/viewform)

https://github.com/omigroup/omigroup/issues/255
